packages/discovery-react-components/.storybook/addons.ts

```diff
@@ -1,3 +1,4 @@
 import '@storybook/addon-knobs/register';
 import '@storybook/addon-actions/register';
 import '@storybook/addon-storysource/register';
```

Comment:
do we still need to register the `addon-storysource` since you removed it from the loader section in the webpack config?